### DISCUSSION TITLE
Footnote-centric academia

### DISCUSSION THREAD

**Discussion by MaPeras** [knowledgefutures/pubpub#1861] [Support]
Hi PubPub-Team,

Within German legal science, citations are usually not in-text-citations. Everything happens within footnotes and looks like that, if you want to cite a book:
[1] Harvey, Rebel Cities, 2019, p. 30.
[2] Harvey, Rebel Cities, 2019, p. 103.

However, when I try to add a citation with the "citation"-function within PubPub I somehow cannot specify the specific page within a book. If I add the page via rich-text, it appears below the rendered citation:
![Bildschirmfoto_2022-03-11_07-07-25](https://user-images.githubusercontent.com/49619613/157812476-e38cdb4f-bac6-4a86-a169-3e3d97e83841.png)

Additionally, footnotes and citations seem to be different things in PubPub. But we just love to add comments within footnotes, that provide further information (and not citations). If I try to do that with your footnote function, I get separated footnote/citation-lists. In standard german-legal-academic texts comments and citations would appear below each other all within footnotes, e.g.:
[1] Harvey, Rebel Cities, 2019, p. 30.
[2] I do not agree with Harvey on this particular issue.
[3] Harvey, Rebel Cities, 2019, p. 103.

We could of course just use your Rich-Text-editing within footnotes every time, but we like the idea of storing more metadata within citations, that could be changed, if our citation style changes.

Thanks!
Max

**Comment by gabestein** [comment]
Hi Max,

If I understand correctly, I think adding another line of bibtex, `pages = {30}` should accomplish what you're looking for in most styles. As for mixing citations with notes, there is a bibtex notes field for exactly this, but very few CSL stylefiles support it. But if, as I think is the case, you're developing your own style file, you can support it and then use it with PubPub (and actually, I haven't tested anything beyond notes, but I'm pretty sure you can basically support arbitrary fields). Details here: https://github.com/jgm/pandoc-citeproc/issues/180

Hope that's helpful!

**Reply by tosteiner** [reply]
Re: first issue, I can confirm this works with the additional `pages = {xx}` line, these can include specific pages or page ranges, e.g. `pages = {116-129}`

**Comment by MaPeras** [comment]
Yes, that's very helpful, thanks a lot! 

You mean adding a note, if I want to add additional text _within_ a citation, like
[1] **For a good argument regarding cities see** Harvey, Rebel Cities, 2019, p. 30.
[2] Harvey, Rebel Cities, 2019, p. 35.?

When I wrote "adding notes within footnotes" I also referred to adding footnotes without any bibliographic data, just comments that do not belong in the main text. Would it be possible to use the "citation"-function for that? 

Cheers
Max

**Reply by gabestein** [reply]
Sorry for the delay! 

If I understand correctly, I think you have two options here. 

For mixing notes and citations as in your first example, you can build your style file to support the bibtex `notes` field, and display those notes however you want.

If you want to have items that are just notes, it probably makes the most sense just to use PubPub’s rich text field. The main reason to use structured citations is so that they can be put in metadata and deposited to services like crossref. But if you want to, I think you could probably just create a bibtex item that only has a notes field. But I’m not 100% sure if that’s valid bibtex, and unfortunately don’t currently have access to a computer where I could test.

**Comment by MaPeras** [comment]
> For mixing notes and citations as in your first example, you can build your style file to support the bibtex notes field, and display those notes however you want.

Good idea, sounds like a valid workaround. 

In the long term it would be great, to change the output of structured citations manually within a citation/footnotes. If you do not use in-text-citations, specific locators need to be added to every footnote. The "pages" line in bibtex (as suggested by @tosteiner ) doesn't work for that, because in some citation styles it is needed to specify the first and/or last page of an article. Using the "notes"-line is also a bit complicated for our authors (we work in large collaborative projects with people used to Office Word) if you just want to add a comment regarding the citation. 

Zotero Word or Libre Office Plugins enable users to add a prefix or suffix to a citation generated through the Zotero database. Having that within PubPub would solve all the above mentioned problems. 

Cheers
Max

**Reply by gabestein** [reply]
Thanks for the continued feedback and research here. It would be great if you could share a screenshot or two of how you're using the Zotero plugins, and what results you expect, so we can get a better sense of exactly what you're looking for.

Digging in a bit, it looks like prefix and suffix are features of citeproc/csl-json, not bibtex (which makes sense, of course, since bibtex is for storing references, but not citations of those references). Once I have a better sense what you're getting from Zotero, I'll need to chat with the engineers about how we're generating references...but it doesn't seem out of the question that we could support those fields more or less natively down the line.

**Reply by MaPeras** [reply]
Hi Gabe,

lost track of this issue for a while. Please find attached some screeshots, how we use the Zotero-Plugin.

![IMG_1295](https://user-images.githubusercontent.com/49619613/194939731-e3496657-bdeb-4806-b3ba-c6ec289bfd2c.jpg)


![Screenshot from 2022-10-10 21-25-57](https://user-images.githubusercontent.com/49619613/194939639-ed1e397d-ccc1-4706-8159-a3d774eebe44.png)
![Screenshot from 2022-10-10 21-26-24](https://user-images.githubusercontent.com/49619613/194939698-3636f317-bfc3-4c1f-8904-7684e8ae3df4.png)

**Reply by kyledenlinger** [reply]
Off-topic, but a robust Zotero integration in the Pub editor would be _amazing_. I can imagine lots of scholarly workflows would be vastly improved and PubPub might look a little shinier next to commercial publishing platforms.
